src/components/Post.jsx

```diff
@@ -53,7 +58,35 @@ function Post(prop) {
     return 6;
   };
   const postHasImages = postImages.some((image) => image.length !== 0);
+ 
+  async function likeshandler() {
+    if (user && likecount !== undefined) {
+
+      const ind = likecount.indexOf(user.email);
+      if (ind !== -1) {
+        likecount.splice(ind, 1);
+        setLikesno(likesno-1);
+      }
+      else {
+        likecount.push(user.email);
+        setLikesno(likesno+1);
```

Comment:
This code might be buggy. Kindly use an updater function
```
setLikesno((currLikesNo) => {
    return currLikesNo + 1
})

Reply:
it is a use state hook which help in showing likes count fastly . I dont think it is having any issue